src/amber/controller/error.cr

```diff
@@ -43,5 +37,13 @@ module Amber::Controller
         message
       end
     end
+
+    private def exception_error_message
```

Comment:
If we merge @marksiemers PR the changes here can be discarded

Reply:
I do think this should be fixed in a different PR.